.pylintrc

```diff
@@ -0,0 +1,31 @@
+[MAIN]
+py-version = 3.6.0
```

Comment:
@Timmmm I think we could use `py-version` as `3.9.0`.

Reply:
I think we want 3.6.0. According to [this](https://github.com/pylint-dev/pylint/issues/5038) it just hides suggestions for features that are only supported in newer versions of Python, so it should be set to the oldest version we support. In other words if we set it to 3.9.0 it will tell us to do some things that are not supported in 3.6.0, so we can't do them, like using the walrus operator for example.

Reply:
Ah wait I didn't see the discussion about increasing the minimum version to 3.9 ( :tada: ). In that case I agree. I'll update the Pyright version too.

Reply:
Actually, as per the tagged PRs earlier, we would not be using the older versions of `Python` that are deprecated and reached its EOL. Hence, we are only supporting this repository to the latest and the supported versions of `Python`. 
And in #299 , we found that version  `python 3.6` is deprecated and reached its EOL, due to which we agreed not to support versions that are deprectaed by Python community itself.